Orlando Walking Tours/Controllers/LocationListVC.swift

```diff
@@ -120,6 +118,19 @@ class LocationListVC: UIViewController
     }
 
     ////////////////////////////////////////////////////////////
+    // MARK: - Navigation
+    ////////////////////////////////////////////////////////////
+
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?)
+    {
+        if let navController = segue.destinationViewController as? UINavigationController,
+           let vc = navController.topViewController as? CurrentTourVC where segue.identifier == "ShowCurrentTourSegue"
```

Comment:
I missed this in the first look over but there is no call to super.prepareForSegue()